src/components/CreatePost/Location.tsx

```diff
@@ -1,26 +1,30 @@
 import { LocationOnOutlined } from '@mui/icons-material';
```

Comment:
proposal;

```suggestion
import { LocationOnOutlined as LocationOnOutlinedIcon } from '@mui/icons-material';
```
`@mui/icons-material`에서 추출한 `Icon 컴포넌트`라고 컴포넌트 이름을 변경하는 건 어떤가요? 민우님 의견이 궁금합니다. (물론 많은 `@mui/icons-material`을 추출하게 된다면 로직이 길어지겠지만….)

Reply:
저는 딱히 불편하다고 생각한적이 없긴한데
명재님 말듣고 한번 생각할 가치가 있다고 생각합니다.